service/dap/server_test.go

```diff
@@ -216,6 +217,150 @@ func TestLaunchTestRequest(t *testing.T) {
 	})
 }
 
+// Test requests that are not supported and return empty responses.
+func TestNoopResponses(t *testing.T) {
+	var got, want dap.Message
+	runTest(t, "increment", func(client *daptest.Client, fixture protest.Fixture) {
+		newResponse := func(reqSeq int, command string) dap.Response {
+			return dap.Response{
+				ProtocolMessage: dap.ProtocolMessage{Type: "response"},
+				RequestSeq:      reqSeq,
+				Success:         true,
+				Command:         command,
+			}
+		}
+
+		client.InitializeRequest()
```

Comment:
This will need to be taken out once https://github.com/go-delve/delve/pull/1914 goes in, so the request numbers are treated as 1-based without changing all the constants below.

Reply:
#1914 was merged.